doc/profiles.md

```diff
@@ -124,6 +126,11 @@ brew install libmatio
 
 Install matio following the [installation instructions](https://github.com/tbeu/matio/#20-building) present in the repository.
 
+## Dynamics full deps
+This profile is enabled by the `ROBOTOLOGY_ENABLE_DYNAMICS_FULL_DEPS` CMake option.
+
+On windows the installation of `CppAD` is currently disabled because of https://github.com/coin-or/CppAD/pull/63 and https://github.com/coin-or/CppAD/pull/61. The user can however install CppAD with `vcpkg`. 
```

Comment:
The problem is that the cppad installed by vcpkg would not be found by `find_or_build_package` due to the use of `YCM_DISABLE_SYSTEM_PACKAGES`, so for now I just say that cppad is not supported, or say that to use the cppad's of vcpkg you need to manually set `YCM_DISABLE_SYSTEM_PACKAGES` to `OFF` .